HW_1/Task_2.py

```diff
@@ -0,0 +1,16 @@
+# Задача 2:
+# Петя, Катя и Сережа делают из бумаги журавликов.
+# Вместе они сделали S журавликов.
+# Сколько журавликов сделал каждый ребенок, если известно,
+# что Петя и Сережа сделали одинаковое количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+# 6 -> 1 4 1
+# 24 -> 4 16 4
+# 60 -> 10 40 10
+
+number = int(input("Сколько всего журавликов сделали дети: "))
+if number % 3 != 0 or number % 2 != 0:
+    print("Некорректный ввод")
+else:
+    numberP_S = int(number/3/2)
```

Comment:
Зачем же использовать функцию int, когда у нас есть прекрасный оператор целочисленного деления?
```suggestion
    numberP_S = number // 3 // 2
``` 
или сразу
```suggestion
    numberP_S = number // 6
```